community.md

```diff
@@ -2,54 +2,61 @@
+  * [Blazor Doughnut Chart](https://github.com/Lupusa87/BlazorDoughnutChartComponent) &ndash; Doughnut charting with gradient colors using SVG.
+  * [Blazor Game Snake](https://github.com/Lupusa87/BlazorGameSnake) &ndash; 2D game snake with customizations, path finding algorithm, and sound effects using SVG.
+  * [Blazor Password Pattern](https://github.com/Lupusa87/BlazorPasswordPattern) &ndash; Password pattern based on SVG.
+  * [Blazor SVG Helper](https://github.com/Lupusa87/BlazorSvgHelper) ([nuget.org](https://www.nuget.org/packages/BlazorSvgHelper/)) &ndash; Create SVG elements with children (circle, rectangle, image, text, and others) and render with RenderTreeBuilder.
+* [Toss](https://github.com/RemiBou/Toss.Blazor) &ndash; A Twitter-like web app that allows the user to sign in, post a new message (a "toss") with hashtag, and select favorite hashtags for finding messages.
 * [TrendTv](https://github.com/MattMarked/TrendTv) ([Demo](http://zaptube2.azurewebsites.net/)) &ndash; Fetch a video list from YouTube and "zap" in real time between trending videos, filtered by country or category.
 
 ## Tooling
 
-* [Razor+ Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=austincummings.razor-plus)
+* [Razor+ Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=austincummings.razor-plus) &ndash; A Visual Studio Code extension that offers improved Razor support.
 
 ## Libraries
 
 * [BlazorBits](http://blazorbits.net) ([GitHub repo](https://github.com/blazorbits)) &ndash; Components including [Monaco editor](https://github.com/Microsoft/monaco-editor).
-* [BlazorComponents (ChartJS Interop)](https://github.com/muqeet-khan/BlazorComponents)
+* [BlazorComponents](https://github.com/muqeet-khan/BlazorComponents) &ndash; A demonstration of [Chart.js](https://github.com/chartjs/Chart.js) interop.
 * [BlazorContextMenu](https://github.com/stavroskasidis/BlazorContextMenu) &ndash; A context menu for blazor.
 * [BlazorDB](https://github.com/chanan/BlazorDB) &ndash; In-memory, persisted to local storage, database for Blazor.
-* [BlazoredLocalStorage](https://github.com/chrissainty/BlazoredLocalStorage)
+* [BlazoredLocalStorage](https://github.com/chrissainty/BlazoredLocalStorage) &ndash; A library to provide access to local storage in Blazor apps.
 * [Blazorous](https://github.com/chanan/Blazorous) &ndash; Maintainable CSS with Blazor.
-* [BlazorStrap (Bootstrap 4 Components)](https://github.com/chanan/BlazorStrap)
-* [Blazor Extensions](https://github.com/BlazorExtensions/)
-* [Blazor local and session storage support](https://github.com/cloudcrate/BlazorStorage)
+* [BlazorStorage](https://github.com/cloudcrate/BlazorStorage) &ndash; Blazor local and session storage support.
+* [BlazorStrap](https://github.com/chanan/BlazorStrap) &ndash; Bootstrap 4 Components for Blazor.
+* [Blazor Extensions](https://github.com/BlazorExtensions/) &ndash; Curated extensions for Microsoft ASP.NET Core Blazor.
```

Comment:
I think we should break out the individual extensions in the BlazorExtensions org so they are easier to discover here. For example, I'd like the SignalR extension to appear directly in this list. @galvesribeiro What do you think?

Reply:
Looks good to me @danroth27 I was thinking the same.

I'll update all README.md files to point to the Home repo and also make it a gh-pages with overviews of all components in it. 

Just got back from vacation so, I'm still slow :P